src/registrar/fixtures.py

```diff
@@ -77,8 +77,8 @@ class UserFixture:
     STAFF = [
         {
             "username": "319c490d-453b-43d9-bc4d-7d6cd8ff6844",
-            "first_name": "Rachid-Analyst",
-            "last_name": "Mrad-Analyst",
+            "first_name": "Rachid-Analysta",
+            "last_name": "Mrad-Analysta",
```

Comment:
is this like a spanglish analyst?

Reply:
This is me using a legit branch and PR to experiment on the pipeline. Disregard, I will clean up.